Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM php:7.0-apache
+
+MAINTAINER Omar Padron "omar.padron@kitware.com"
+
+RUN apt-get update                                                             \
+ && apt-get install -y gnupg                                                   \
+ && curl -sL https://deb.nodesource.com/setup_6.x | bash                       \
+ && apt-get install -y git libbz2-dev libfreetype6-dev libjpeg62-turbo-dev     \
+    libmcrypt-dev libpng-dev libpq-dev libxslt-dev libxss1 nodejs unzip wget   \
+    zip                                                                        \
+ && docker-php-ext-configure pgsql --with-pgsql=/usr/local/pgsql               \
+ && docker-php-ext-configure gd --with-freetype-dir=/usr/include/              \
+                                --with-jpeg-dir=/usr/include/                  \
+ && docker-php-ext-install -j$(nproc) bcmath bz2 gd pdo_mysql pdo_pgsql xsl    \
+ && pecl install xdebug-2.5.5                                                  \
```

Comment:
@opadron why xdebug if this is for production environments?

Reply:
Thanks for catching this.  I'll remove it.